package-build.el

```diff
@@ -589,17 +581,33 @@ Return a cons cell whose `car' is the root and whose `cdr' is the repository."
         (package-build--princ-checkout repo dir)
         (package-build--run-process nil "hg" "clone" repo dir)))
       (if package-build-stable
-          (let* ((bound (goto-char (point-max)))
-                 (tag-version (and (package-build--run-process dir "hg" "tags")
-                                   (or (package-build--find-tag-version-newest
-                                        "^\\(?:v[.-]?\\)?\\([0-9]+[^ \t\n]*\\)[ \t]*[0-9]+:\\([[:xdigit:]]+\\)$"
-                                        bound
-                                        2)
-                                       (error
-                                        "No valid stable versions found for %s"
-                                        name)))))
-            (package-build--run-process dir "hg" "update" (nth 2 tag-version))
-            (cadr tag-version))
+        (let ( (bound (goto-char (point-max)))
+               (regexp (or (plist-get (package-build--config name) :version-regexp)
```

Comment:
Oh, I changed the one in `package-build--checkout-git` and forgot to do the same here.